src/testHelper.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiClient } from "./api";
+import { Question } from "./types";
+
+let nextId = 0;
+
+const getNextId = (): number => {
+    const value = nextId;
+    nextId += 1;
+    return value;
```

Comment:
Only a small thing - this can be implemented like that

```suggestion
    return nextId++;
```